src/main/java/ca/bc/gov/notificationservice/controller/UpdateCardController.java

```diff
@@ -26,8 +26,12 @@
   @Autowired
   TeamsChannelService teamsChannelService;
 
-  @PostMapping(value = "update", produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<String> update(@RequestBody NotificationBody teamsUpdate) {
+  @PostMapping(value = "update/{token}", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<String> update(@PathVariable("token") String token, @RequestBody NotificationBody teamsUpdate) {
+      if (!notificationServiceProperties.getTokens().contains(token)) {
```

Comment:
what happens if notificationServiceProperties.getTokens().contains(token)

Reply:
If the token exists in the list, good to keep goin.